packages/yarnpkg-core/sources/Project.ts

```diff
@@ -184,9 +184,18 @@ export class Project {
    */
   public peerRequirements: Map<string, PeerRequirement> = new Map();
 
+  /**
+   * Contains whatever data the installers (cf `Linker.ts`) want to persist
+   * from an install to another.
+   */
   public installersCustomData: Map<string, unknown> = new Map();
 
+  /**
+   * Those checksums are used to detect whether the relevant files actually
+   * changed since we last read them (to skip part of their generation).
+   */
```

Comment:
This will only be shown for `lockFileChecksum` making the comment not mean much

Reply:
I've checked how to spread comments over multiple fields but didn't find a good way. Still useful when reading the sources.